CODING_STANDARDS.md

```
@@ -1 +1 @@
-You can find the C# coding standards for the OSP Suite in the developer documentation [here](https://github.com/Open-Systems-Pharmacology/developer-docs/blob/main/setup/coding_standards.md).
\ No newline at end of file
+You can find the C# coding standards for the OSP Suite in the developer documentation [here](https://dev.open-systems-pharmacology.org/setup/coding_standards).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)